packages/viewer/src/components/viewer/viewer.css

```diff
@@ -27,7 +27,7 @@
   width: 100%;
   height: 100%;
   position: relative;
-  background: var(--image-background, var(--viewer-background, #ffffff));
+  background: #ffffff;
```

Comment:
Is this still used as the default? Or can it be removed altogether?

Reply:
Let me take a look! This was required with an initial approach, but after converting the fallback to just `#ffffff`, this may be possible to remove

Reply:
Looks like it's no longer required - removed with [4765024](https://github.com/Vertexvis/vertex-web-sdk/pull/533/commits/4765024acdbb22d03c3221fcfc04639d474e1ac6). Good catch!